.rhdh/docker/Dockerfile

```diff
@@ -229,6 +229,11 @@ ENV CHOKIDAR_USEPOLLING='1' CHOKIDAR_INTERVAL='10000'
 # To avoid running scripts when using `npm pack` to install dynamic plugins
 ENV NPM_CONFIG_ignore-scripts='true'
 
+# Segment key for production
+#TODO
+ENV SEGMENT_WRITE_KEY=""
+ENV SEGMENT_TEST_MODE=false
```

Comment:
Would we want this `true` for CI builds from main branch?

Reply:
I thought that CI builds are using `docker/Dockerfile`

Reply:
yes, but do we want test mode enabled for any of the builds? eg., maybe for the janus builds it's test mode = true, and for the downstream builds it's test mode = false?

Reply:
good point. TBH I'm still not sure about this one :-( 
I'll change it to `true` for no to have it disabled for CI builds.

But in this case
https://gitlab.cee.redhat.com/rhidp/rhdh/-/merge_requests/10/diffs?commit_id=35426e292eda191d0b3ecf649695ecc416ff73a7#86d2a35e6325bba5ceaf29e203b14cd809af55fd_175_184 needs to be updated to also switch `SEGMENT_TEST_MODE` to `false` for release branches